packages/components/addon/components/hds/button-set/index.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+// import { assert } from '@ember/debug';
+
+export default class HdsButtonSetIndexComponent extends Component {
+  /**
+   * Get the class names to apply to the component.
+   * @method classNames
+   * @return {string} The "class" attribute to apply to the component.
+   */
+  get classNames() {
+    let classes = ['hds-button-set'];
```

Comment:
when there is no logic attached to the classnames generation, we write directly the class in the HBS file (see for example https://github.com/hashicorp/design-system/blob/ed96740352c8a1bda0cebbcec41eb6eb7de0a67d/packages/components/addon/components/hds/form/indicator/index.hbs#L2)